lib/nfg_ui/engine.rb

```diff
@@ -6,6 +6,15 @@ class Engine < ::Rails::Engine
 
     config.autoload_paths << Engine.root.join("lib")
 
+
+
+    # TODO: Could not get asset paths to precompile the images/nfg_ui
+    # folder. For now, brute force all images via Dir.glob...
+    initializer "nfg_ui.assets.precompile" do |app|
+      images = Dir.glob(Engine.root.join('app', 'assets', 'images', 'nfg_ui', '**', '*'))
+      app.config.assets.precompile += images
+    end
```

Comment:
@ajporterfield is there a way to do this where I simply tell the precompiler to pull in the `assets/images/nfg_ui/**/*` folder... instead of this brute force approach? 

I've tried the following approaches:
https://github.com/network-for-good/nfg_ui/pull/17/commits/7ead295aa3a835c45d851999e40e6d06c96a1b05
https://github.com/network-for-good/nfg_ui/pull/17/commits/7c0f638145371773cfe46d227b3b7198fb162063
https://github.com/network-for-good/nfg_ui/pull/17/commits/7d7e2f12310df76169cd3459f3dcea1f9901d5f5
https://github.com/network-for-good/nfg_ui/pull/17/commits/f7f87b34d723e0447c2db2ea4d933ae02de542fc
https://github.com/network-for-good/nfg_ui/pull/17/commits/8cdcacf5c826045617d9c7faebd0d835be0aee24
https://github.com/network-for-good/nfg_ui/pull/17/commits/db7dbcca489cb54ea4b441a28300de40915bed83
etc...

Thanks!

Reply:
@kcpavan would you actually be the one to take a look at this? @ajporterfield has quite a full plate at the moment -- sorry Andrew!

Reply:
This may be of some interest: https://github.com/rails/sprockets/issues/542

Reply:
@hoenth the resolution proposed here worked for me!

The updated solution looks like this:
On our manifest file (`app/assets/config/nfg_ui_manifest.js`), I needed to "link" the images folder:
```js
//= link_tree ../images/nfg_ui
```

Then, I updated the engine initializer to simply precompile the manifest (which it had never done before, so technically the manifest wasn't being used anyways)

In `lib/nfg_ui/engine.rb` update the precompile initializer to:
```ruby
   initializer "nfg_ui.assets.precompile" do |app|
      app.config.assets.precompile += %w( config/nfg_ui_manifest.js )
    end
```

Way better. Thank you very much!